src/main/java/seedu/eduke8/command/AnswerCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.eduke8.command;
+
+import seedu.eduke8.common.DisplayableList;
+import seedu.eduke8.option.Option;
+import seedu.eduke8.question.Question;
+import seedu.eduke8.ui.Ui;
+
+public class AnswerCommand extends Command {
+    private Option option;
+    private Question question;
+
+    public AnswerCommand(Option option, Question question) {
+        this.option = option;
+        this.question = question;
+    }
+
+    @Override
+    public void execute(DisplayableList optionList, Ui ui) {
```

Comment:
Since optionList is unused here, do you think the execute method's parameters in the Command abstract class can be changed?

Reply:
Just a suggestion, but I think what you have now works too! The suggestion is just something to think about for future implementations!